_episodes/09-cmorization.md

```diff
@@ -0,0 +1,1053 @@
+> > of the variable is ``land`` and that the frequency is ``mon``. This
+> > means we have to check the CMOR table "Lmon" for an entry. We focus
+> > our search on the CMIP6 tables since these are the most recent tables
+> > available. You can find the lists here:
+> > `<https://github.com/ESMValGroup/ESMValCore/tree/master/esmvalcore/cmor/tables/cmip6/Tables>`
+> >
+> > We do not find the variable "cVegStderr" in the "Lmon" table, which
+> > means we will have to write our own custom table for this.
+> > There were two examples given which we could use as templates for the
+> > new custom table that we have to create. These two examples can be
+> > found here: [cVeg](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/cmip6/Tables/CMIP6_Lmon.json)
+> > and [gppStderr](https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/CMOR_gppStderr.dat)
+> >
+> > We have to create a new file with the name ``CMOR_cVegStrerr.dat`` in
+> > the custom CMOR table folder (https://github.com/ESMValGroup/ESMValCore/blob/master/esmvalcore/cmor/tables/custom/).
+> > The content of the file should then look like this:
+> >
+> > ```
+> > SOURCE: CMIP6
+> > !============
+> > variable_entry:    cVegStderr
+> > !============
+> > modeling_realm:    land
+> > !----------------------------------
+> > ! Variable attributes:
+> > !----------------------------------
+> > standard_name:
+> > units:             kg m-2
+> > cell_methods:      area: mean where land time: mean
+> > cell_measures:     area: areacella
+> > long_name:         Carbon Mass in Vegetation Error
+> > !----------------------------------
+> > ! Additional variable information:
+> > !----------------------------------
+> > dimensions:        longitude latitude time
+> > out_name:          cVegStderr
+> > type:              real
+> > !----------------------------------
+> > !
+> > ```
+> >
+> > Note that there is no entry for ``standard_name``. This is on purpose.
+> > It is a sign for the ESMValTool to not crash although the variable that
+> > we are looking for is ok to have no official CMIP6 ``standard_name``.
+> >
+> {: .solution}
+{: .challenge}
+
+## Create a CMORizer for the new dataset
+
+Now that we have the data ready, our first assumption would be that the data
+is already following the CMOR standard. So we store the file in the "OBS6"
+folder of our data path, and rename it manually to an ESMValTool readable
+name. For our example of the "FLUXCOM" data that we downloaded (which is
+from the year 2000) the name of the dataset would then be:
+
+```bash
+OBS6_FLUXCOM_reanaly_ANN-v1_Lmon_gpp_200001-200012.nc
+```
+
+We also have made sure that our variable of choice is listed either
+on a pre-existing or custom CMOR table. So now we are ready to test
+if the data is actually following the necessary CMOR standard already,
+or if we have to do some reformatting for the dataset.
+
+> ## Run the test recipe again
```

Comment:
These sections are not shown correctly on the tutorial webpage. Please preview the lesson locally and fix it.